pixi_diff_to_markdown/cli.py

```diff
@@ -46,7 +47,11 @@ def main(
     ] = None,
 ):
```

Comment:
```suggestion
) -> int:
```

and `return 0` at the end?

Reply:
Done!